esp32/modules/easyrtc.py

```diff
@@ -7,8 +7,11 @@
 import machine, time
 
 # Functions
-def string(date=False, time=True):
-    [year, month, mday, wday, hour, min, sec, usec] = machine.RTC().datetime()
+def string(print_date=False, print_time=True, timestamp = -1):
+    if timestamp<0:
+        [year, month, mday, wday, hour, minute, sec, usec] = machine.RTC().datetime()
+    else:
+        [year, month, mday, hour, minute, second, wday, yday] = time.localtime(timestamp)
```

Comment:
shouldn't `second` be `sec` here (or vice-versa)?

pretty confusing that these 2 assignments assign to different variables, might be clearer to rename `usec` to `unused_usec` and `yday` to `unused_yday` to make explicit that those aren't used?